src/Table/index.js

```diff
@@ -25,10 +24,11 @@ Table.Body = ({ children, className }) => (
   </tbody>
 );
 
-Table.HeaderCell = ({ children, className }) => (
+Table.HeaderCell = ({ children, className, colSpan }) => (
   <th
     tw="px-6 py-2 bg-white text-left text-xs leading-5 font-medium text-foreground uppercase tracking-wider"
     className={className}
+    colspan={colSpan}
```

Comment:
```suggestion
    colSpan={colSpan}
```

Attributes need to be camelCase. Although this works, it throws a warning.